### DISCUSSION TITLE
Is there any way to make these automations headless?

### DISCUSSION THREAD

**Discussion by masautt** [AutomaApp/automa#34] [Q&A]
Hoping to export these workflows from the extension to a server

**Comment by Kholid060** [comment]
If you mean to export the workflow so it can run on puppeteer, playwright, or selenium. I don't think it's possible right now

**Reply by masautt** [reply]
Yes puppeteer would be amazing!

**Reply by ambirex** [reply]
I was just looking into the export file format and its just JSON and not too complicated, at least from the workflow I exported. There is a good chance someone could copy the execution engine and port it to puppeteer.

**Reply by Kholid060** [reply]
I don't think I will add this feature, but a pull request is welcome if someone wants to.

**Reply by ambirex** [reply]
I don’t think a headless runner would make sense in this repo. I would imagine it should likely be its own package (something like `automa-headless-runner`)

**Reply by Kholid060** [reply]
Creating its own package actually a good idea, but I can't do another side project right now because of the bugs and feature requests that currently automa has.

**Reply by ambirex** [reply]
I might take a crack at it in the next months or so as a side project during some of my time off, @Kholid060 do you have any reservations if I call it `automa-headless-runner` or would you prefer I not use the name `Automa`?

**Reply by Kholid060** [reply]
It's actually up to you.

**Reply by Kholid060** [reply]
But I think you should add `automa` in it, so the user easily find and know what the package is for
